src/Main/MainInterface.jsx

```diff
@@ -16,11 +16,13 @@ const STYLE = { width: '20%' }
 
 function MainInterface(props) {
   const { service } = props
-  const [state] = useActor(service)
   const mainUIGroup = useRef(null)
 
+  let use2D = useSelector(service, (state) => state.context.use2D)
+  let uiGroups = useSelector(service, (state) => state.context.uiGroups)
+
   useEffect(() => {
-    state.context.uiGroups.set('main', mainUIGroup.current)
+    uiGroups.set('main', mainUIGroup.current)
```

Comment:
I looks like we add many entries to this `uiGroups` Map (aka Dictionary).  But we never `uiGroups.get`  I checked the itk-vtk-viewer repo and we only `uiGroups.get` in under the `UI/reference-ui` directory.  But we are completely over-riding the code in the main repos `UI/reference-ui` directory (via the `materialUIMachineOptions.js` file).  

So lets remove all the uiGroups.sets we are doing throughout the code.